src/h5web/vis-packs/core/shared/Html.tsx

```diff
@@ -92,15 +119,16 @@ const Html = forwardRef<HTMLDivElement, HtmlProps>((props, ref) => {
       </div>,
       el
     );
-  }, [children, className, el, style, ref, divProps]);
+  }, [children, className, divProps, el, getInnerDivTransform, style]);
 
+  // Update inner `div` position on every frame
   useFrame(() => {
-    if (followCamera) {
-      updateHtmlPosition();
+    if (innerRef.current) {
+      innerRef.current.style.transform = getInnerDivTransform();
     }
```

Comment:
The condition checks whether the ref is ready, which makes `useFrame` safe if called before `useEffect`. Note that if `followCamera` is disabled, the `transform` is set to the empty string (i.e. the CSS property is not set, or removed if present).